Makefile

```diff
@@ -7,7 +7,7 @@ install: grpc-install mockgen-install goimports-install update-proto
 proto: grpc goimports proxy grpc-mock copyright
 
 # Update submodule and compile proto files.
-update-proto: update-proto-submodule proto update-dependencies gomodtidy
+update-proto: update-proto-submodule gomodtidy proto update-dependencies
```

Comment:
`gomodtidy` should come before `proto`!

Reply:
No it should come after. This will cause errors as we're generating packages we require to run tidy